hr_webcam/hr.py

```diff
@@ -0,0 +1,24 @@
+from openerp.osv import osv
```

Comment:
This file defines hr_employee, it would be more helpful for maintenance to name the file `hr_employee.py`